src/main/webapp/WEB-INF/tags/instructor/results/sessionPanel.tag

```diff
@@ -21,8 +21,10 @@
             <label class="col-lg-4 control-label">Session:</label>
             <div class="col-lg-8">
               <p class="form-control-static">${sessionPanel.feedbackSessionName}
-                <c:if test="${not empty sessionPanel.editLink}">
-                  <a href="${sessionPanel.editLink}">[Edit]</a>
+                <c:if test="${not empty sessionPanel.editLink}">&nbsp
+                  <a href="${sessionPanel.editLink}">
+                    <span class="glyphicon glyphicon-edit" data-original-title="Edit feedback session details" data-placement="top" data-toggle="tooltip" id="edit_icon_button"></span>
```

Comment:
I see that you changed it to the content of `Const.Tooltips.FEEDBACK_SESSION_EDIT`. So why not reference the constant here directly

```jsp
data-original-title="<%= Const.Tooltips.FEEDBACK_SESSION_EDIT %>"
```

Reply:
If you do like this with same indentation, you will follow the correct indentation style and also will not have to add an explicit space

<img width="685" alt="screen shot 2018-07-02 at 4 28 27 pm" src="https://user-images.githubusercontent.com/12574756/42153399-94a12acc-7e15-11e8-8ca7-3d9e09bb744d.png">

Reply:
I attempted to do something in this nature, but I didn't know how.